src/containers/Sidebar/Sidebar.test.jsx

```diff
@@ -88,6 +89,25 @@ describe('<Sidebar />', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('renders correctly when subscriptionManagementScreen is false', () => {
+    // should cause both subscription management and enrollment links to not be present
+    const store = mockStore({
+      sidebar: {
+        ...initialState.sidebar,
+      },
+      portalConfiguration: {
+        enableSubscriptionManagementScreen: false,
+      },
+    });
+
+    const tree = renderer
```

Comment:
I don't think we should add more snapshot tests if we can avoid it. I'd greatly prefer to render the component and verify that the link text isn't there.

Reply:
big plus 1

Reply:
but also okay if we want to ticket it separately in case we think we lose coverage by not having this test